### ISSUE TITLE
React 0.13 warning about mutated props

### ISSUE THREAD

**Issue by neverfox** [tonsky/rum#35]
I could just be doing something wrong but Rum seems to be running afoul of the change in React 0.13 to immutable props. See [here](https://github.com/zetachang/react.rb/issues/18) and [here](https://groups.google.com/forum/#!topic/reactjs/A9Ul3TXlaOA).

``` clojure
(defonce app-state (atom {:text "Hello World!"}))

(rum/defc app < rum/cursored rum/cursored-watch [app-state]
          [:h1 (:text @app-state)])

(defn main []
  (rum/mount
    (app app-state)
    (. js/document (getElementById "app"))))
```

```
Warning: Don't set .props.:rum/state of the React component <app />. Instead, specify the correct value when initially creating the element or use React.cloneElement to make a new element with updated props.
```

**Comment by tonsky** [comment]
I have to update Rum for 0.13, some things might not be working

**Comment by neverfox** [comment]
I see. Thanks. Not trying to nag, but do you have a timeline for that update?

**Comment by tonsky** [comment]
No timeline, sorry

**Comment by tonsky** [comment]
Should be fixed in 0.3.0
